WPFHospitalEditor/Model/TimeInterval.cs

```diff
@@ -14,5 +15,15 @@ public TimeInterval(DateTime start, DateTime end)
         public DateTime Start { get; set; }
         public DateTime End { get; set; }
         public TimeSpan Duration => End - Start;
+
+        public bool IsValid()
+        {
+            if (Start >= End)
+            {
+                MessageBox.Show("End time must be after start time!", "");
+                return true;
+            }
+            return false;
```

Comment:
Method is called `IsValid` and returns true if start is after end o.O You should switch true and false here.